class ExistingUser(Exception):
    ...


class AuthenticationError(Exception):
    ...
